consultation_analyser/support_console/jinja2/support_console/ml_pipeline_test.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/notification-banner/macro.html' import govukNotificationBanner -%}
+{%- from 'govuk_frontend_jinja/components/warning-text/macro.html' import govukWarningText -%}
+
+
+{% block content %}
+
+  {% for message in messages %}
+    {% if message.tags == "info" %}
+      {{ govukNotificationBanner({
+        'html': message
+      }) }}
+    {% elif message.tags == "warning" %}
+      {{ govukWarningText({
+        'text': message,
+        'iconFallbackText': "Warning"
+      }) }}
+    {% endif %}
+  {% endfor %}
+
+  <h1 class="govuk-heading-xl">Test the machine learning pipeline</h1>
+    <form method="post" novalidate>{{ csrf_input }}
```

Comment:
We can also crispy-formsify this, but perhaps can do after merge

Reply:
Not sure exactly what this means - hopefully we can discuss at some point! (Maybe build on this once merged.)